app/templates/partials/answers/date.html

```diff
@@ -1,27 +1,29 @@
 {% set subfields %}
- <div class="fieldgroup fieldgroup--date" data-qa="widget-date">
+ <div class="fieldgroup fieldgroup--inline fieldgroup--date" data-qa="widget-date">
   <div class="fieldgroup__fields">
     {% if form.fields[answer['id']]['day'] %}
-    <div class="field field--input field--day">
-      <label class="label mercury" for="{{answer['id']}}-day" id="label-{{answer['id']}}-day" data-qa="label-day">{{ _("Day") }}</label>
+    <div class="field field--input">
+      <label class="label u-fs-r" for="{{answer['id']}}-day" id="label-{{answer['id']}}-day" data-qa="label-day">{{ _("Day") }}</label>
       {% set input = form.fields[answer['id']]['day'] %}
-      {% set input_placeholder = "DD" %}
+      {% set width_class = "input--w-2" %}
+      {% set type = "number" %}
```

Comment:
This should be changed to create an input dict as we do with other input types. This would tidy up the changes to `input.html` (`width_class` and `input.type` are not needed). Something like:
```python
{% set input = {
    'type': 'number',
    'classes': 'input--w-2',
    'data': input._value(),
    'name': answer.id,
    'id': answer.id,
    'placeholder': 'DD'
} %}
```

Also, are the placeholders supposed to be removed?

Reply:
Placeholders should be avoided as they are bad for accessibility so I've removed them to match the pattern library.

How would I get the correct name/id/value for each field in for the date answer?